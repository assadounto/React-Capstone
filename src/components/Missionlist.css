.mission-container {
  border: 1px solid #d8d8d8;
}

.headings {
  display: grid;
  grid-template-columns: 1fr 5fr 1fr 1fr;
  border: 1px solid #d8d8d8;
  padding: 10px;
}
