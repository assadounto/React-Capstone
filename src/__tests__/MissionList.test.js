import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/ConfigureStore';
import Missionlist from '../components/Missionlist';

it('Rocket render correctly', () => {
  const Tree = TestRenderer.create(
    <Provider store={store}>
      <Missionlist />
    </Provider>,
  );
  expect(Tree).toMatchSnapshot();
});
