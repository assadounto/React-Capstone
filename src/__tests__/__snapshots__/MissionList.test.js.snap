// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket render correctly 1`] = `
<div
  className="mission-container"
>
  <div
    className="headings"
  >
    <div
      className="mission-heading box-1"
    >
      <h3>
        Mission
      </h3>
    </div>
    <div
      className="mission-description-headline box-2"
    >
      <h3>
        description
      </h3>
    </div>
    <div
      className="mission-status box-3"
    >
      <h3>
        status
      </h3>
    </div>
    <div
      className="mission-blank box-3"
    />
  </div>
  <div
    className="mission-items"
  />
</div>
`;
